src/Plugins/ComplexCore/test/QuickSurfaceMeshFilterTest.cpp

```diff
@@ -15,7 +15,8 @@
 using namespace complex;
 using namespace complex::UnitTest;
 using namespace complex::Constants;
-
+namespace SurfaceMeshingTest
+{
 std::shared_ptr<DataStructure> CreateEbsdTestDataStructure()
```

Comment:
```suggestion
DataStructure CreateEbsdTestDataStructure()
```
Doesn't need to be `std::shared_ptr`.